target/Target/AntiPattern/Stan0206.hs

```diff
@@ -20,3 +20,10 @@ data PlainExample
         Int
     | Mk2 !Int
     | Mk3 Bool
+
+data ConstraintExample
+    = forall a .
+        Constr a
+    | forall a b .
+        Constr2 a
+          !b
```

Comment:
Can we add two more examples?

1. With constraint, and not only with `forall`
2. With `forall` and the record data type (similar to the original issue data type)